lib/files.c

```diff
@@ -251,15 +252,15 @@ bool files_load_tpm_context_from_file(TSS2_SYS_CONTEXT *sapi_context,
      */
     UINT32 version;
     TPMS_CONTEXT context;
-    bool result = files_read_header(f, &version);
+    bool result = files_read_header(fstream, &version);
     if (!result) {
         LOG_WARN(
-                "The tpm context file \"%s\" does not appear in the proper format, assuming old format, this will be converted on the next save.",
-                path);
-        rewind(f);
-        result = files_read_bytes(f, (UINT8 *) &context, sizeof(context));
+            "The loaded tpm context does not appear in the proper format,"
```

Comment:
Is this sentence correct, shouldn't be "does not appear to be in the proper format" ?

Reply:
Yes it should